packages/import/test/ValidateCsv.spec.ts

```diff
@@ -0,0 +1,307 @@
+        command.parse([ "csv1.csv", "csv2.csv" ], parseOptions);
+    })
+})
+
+const CSV_WITHOUT_FILE = `ID,DESCRIPTION,TERMS_AND_CONDITIONS TYPE,TERMS_AND_CONDITIONS PARAMETERS
+1,An NFT ID,none,none
+2,Another NFT ID,none,none,
+`;
+
+const CSV_WITHOUT_FILE_EMPTY_ROWS = `ID,DESCRIPTION,TERMS_AND_CONDITIONS TYPE,TERMS_AND_CONDITIONS PARAMETERS
+1,An NFT ID,TYPE2,PARAM2
+
+2,Another NFT ID,,
+`;
+
+
+const CSV_WITHOUT_FILE_WRONG_NO_HEADER = `1,An NFT ID
+2,Another NFT ID
+`;
+
+const CSV_WITHOUT_FILE_WRONG_BAD_HEADER = `ID,DESCRIPTIONS
+1,An NFT ID
+2,Another NFT ID
+`;
+
+const CSV_WITH_FILE = `ID,DESCRIPTION,FILE NAME,FILE CONTENT TYPE,FILE SIZE,FILE HASH,TERMS_AND_CONDITIONS TYPE,TERMS_AND_CONDITIONS PARAMETERS
+programming_music.jpg,Programming Music,programming_music.jpg,image/jpeg,90718,0xa025ca5f086f3b6df1ca96c235c4daff57083bbd4c9320a3013e787849f9fffa,CC4.0,BY
+lucas_games_characters.jpg,LucasArts Games Characters,lucas_games_characters.jpg,image/jpeg,91880,0x546b3a31d340681f4c80d84ab317bbd85870e340d3c2feb24d0aceddf6f2fd31,CC4.0,BY-NC
+`;
+
+const CSV_WITH_FILE_AND_TOKEN = `ID,DESCRIPTION,FILE NAME,FILE CONTENT TYPE,FILE SIZE,FILE HASH,TERMS_AND_CONDITIONS TYPE,TERMS_AND_CONDITIONS PARAMETERS,RESTRICTED,TOKEN TYPE,TOKEN ID,TOKEN ISSUANCE
+programming_music.jpg,Programming Music,programming_music.jpg,image/jpeg,90718,0xa025ca5f086f3b6df1ca96c235c4daff57083bbd4c9320a3013e787849f9fffa,CC4.0,BY,Y,owner,0x900edc98db53508e6742723988b872dd08cd09c2,1
+lucas_games_characters.jpg,LucasArts Games Characters,lucas_games_characters.jpg,image/jpeg,91880,0x546b3a31d340681f4c80d84ab317bbd85870e340d3c2feb24d0aceddf6f2fd31,CC4.0,BY-NC,Y,owner,0x900edc98db53508e6742723988b872dd08cd09c2,1
+`;
+
+const CSV_WITH_TOKEN = `ID,DESCRIPTION,TERMS_AND_CONDITIONS TYPE,TERMS_AND_CONDITIONS PARAMETERS,TOKEN TYPE,TOKEN ID,TOKEN ISSUANCE
+programming_music.jpg,Programming Music,CC4.0,BY,owner,0x900edc98db53508e6742723988b872dd08cd09c2,1
+lucas_games_characters.jpg,LucasArts Games Characters,CC4.0,BY-NC,owner,0x900edc98db53508e6742723988b872dd08cd09c2,1
+`;
+
+const CSV_WITH_FILE_WRONG_TYPE = `ID,DESCRIPTION,FILE NAME,FILE CONTENT TYPE,FILE SIZE,FILE HASH,TERMS_AND_CONDITIONS TYPE,TERMS_AND_CONDITIONS PARAMETERS
+programming_music.jpg,Programming Music,programming_music.jpg,wrong/type,90718,0xa025ca5f086f3b6df1ca96c235c4daff57083bbd4c9320a3013e787849f9fffa,CC4.0,BY
+lucas_games_characters.jpg,LucasArts Games Characters,lucas_games_characters.jpg,wrong/type,91880,0x546b3a31d340681f4c80d84ab317bbd85870e340d3c2feb24d0aceddf6f2fd31,CC4.0,BY-NC
+`;
+
+const CSV_WITH_TOKEN_WRONG_TYPE = `ID,DESCRIPTION,TERMS_AND_CONDITIONS TYPE,TERMS_AND_CONDITIONS PARAMETERS,TOKEN TYPE,TOKEN ID,TOKEN ISSUANCE
+programming_music.jpg,Programming Music,CC4.0,BY,wrongType,0x900edc98db53508e6742723988b872dd08cd09c2,1
+lucas_games_characters.jpg,LucasArts Games Characters,CC4.0,BY-NC,wrongType,0x900edc98db53508e6742723988b872dd08cd09c2,1
+`;
+
+
+
+fdescribe("ValidateCsv - readItemsCsv", () => {
```

Comment:
There is a forgotten `fdescribe` here. Furthermore, I think that these "advanced" validation tests, along with their resources, should be moved to the `csv` package.

Reply:
I agree that `CsvValidator` should be unit-tested in its own package, although it way easier to setup tests data formatted in CSV. I'll fix that in a new PR.